code/java/pom.xml

```diff
@@ -38,41 +38,41 @@
         <!-- AWS Cloud Development Kit -->
         <dependency>
             <groupId>software.amazon.awscdk</groupId>
-            <artifactId>cdk</artifactId>
-            <version>0.22.0</version>
```

Comment:
Would be better to export version to <cdk.version>0.36.2.DEVPREVIEW</cdk.version> and ref on it?